src/notation/internal/notationactioncontroller.h

```diff
@@ -230,6 +230,8 @@ class NotationActionController : public actions::Actionable, public async::Async
 
     void registerPadNoteAction(const mu::actions::ActionCode&, Pad padding);
     void registerTabPadNoteAction(const mu::actions::ActionCode&, Pad padding);
+    void notifyAboutActionIsTriggered(const mu::actions::ActionCode&);
+    void notifyAboutActionWasTriggered(const std::string& actionTitle);
```

Comment:
let's rename this method to `notifyAccessibilityAboutVoiceInfo(const std::string& info)`

Reply:
Sure

Reply:
@Eism All updated now.